.github/workflows/spelling.yml

```diff
@@ -0,0 +1,28 @@
+name: Spell checking
+on: [push, pull_request]
+
+jobs:
+  spelling:
+    name: Spell checking
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout-merge
+      if: "contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2
+      with:
+        ref: refs/pull/${{github.event.pull_request.number}}/merge
+    - name: checkout
+      if: ${{ github.event_name == 'push' ||
+        (
+          contains(github.event.comment.body, '@check-spelling-bot apply')
```

Comment:
Now I see why you had `issue_comment` as an event trigger. This workflow should work if someone comments `@check-spelling-bot apply`. Now that you've removed `issue_comment` trigger it won't work.

I'm not sure we need this.

Reply:
Alright, understood. So I have to remove the entire `-name: checkout` block (including the things inside it like `if:` , `uses:` ) right? 

Similarly, I think we don't need this block too: `if: ${{ github.event_name != 'issue_comment' ||` because it depends on the `issue_comment`.

Would love to hear your opinion.

Reply:
Yes, that's exactly what I was suggesting.